### DISCUSSION TITLE
About Reisa's passive skills

### DISCUSSION THREAD

**Discussion by maromameta** [lonqie/SchaleDB#48] [Questions] [answered]
Sorry, this may not be the right place to ask this question, but I didn't know where to inquire, so I'm asking it here.

On SchaleDB, it says that Leisa's passive skill increases HP by 16.24%, and the calculation is based on 16.24%.
However, in the game it states 16.2%.
On students.json it also says 16.2%.
Which is correct?
I have not calculated the attack power, but it may be the same.

**Comment by lonqie** [comment, marked as answer]
In-game skill descriptions (and translations of them) round percentage values to 1 decimal place when < 100% or integers when > 100%.

The values used in calculations and shown on that screen use the raw values which are more precise.
